docs/USAGE.md

```diff
@@ -54,8 +58,56 @@ In order to build test plans, Testground needs to know where its source director
 
 For special cases, supply the `TESTGROUND_SRCDIR` environment variable.
 
+## Starting a testground daemon
+
+---
+
+⚠️ In the past, if the `[client]` section in the `.env.toml` file was not set
+up, we would start an embedded daemon to service the request. To avoid
+confusion, we no longer do that.
+
+---
```

Comment:
We don't need this section. This was the case for barely a week or two, probably nobody even used it that way (as Testground is not really usable right now, so we just generate docs for technical debt that we incurred during iteration.

Reply:
Fine with removing it -- I wanted to be nice to our past citizens.

Reply:
This is still there though.